bool open_gripper
---
bool success
